### ISSUE TITLE
Build error: GetRequestContextForRenderProcess

### ISSUE THREAD

**Issue by alexflint** [electron/brightray_example#29]
I ran

```
./script/bootstrap https://s3.amazonaws.com/github-janky-artifacts/libchromiumcontent
./script/build
```

and got this:

```
vendor/brightray/browser/browser_context.h:51:33: error: 'GetRequestContextForRenderProcess' overrides a member function but is not marked 'override'
```

The libchromiumcontent version was

```
https://s3.amazonaws.com/github-janky-artifacts/libchromiumcontent/osx/64bit/3f108160eded696b3e843c2fbb4e5e2ac696555e/libchromiumcontent.zip
```

**Comment by alexflint** [comment]
Looks like this was fixed in brightray, but brightray_example is still pointing to a slightly older version of brightray

**Comment by pfrazee** [comment]
Yeah, the repo will need an update. I updated the submodule and got new compile errors. When I have the time, if nobody else has done it yet, I'll do it myself

**Comment by pfrazee** [comment]
Ok, I'm doing it now because I want to toy with the example build. After updating to latest, this change breaks the example: https://github.com/electron/brightray/commit/e26f8073df2b20ba2169ffb082c5f135d542313b#diff-05af3a10d22f96957eeeca8001a50581. Solving...

**Comment by pfrazee** [comment]
In addition to needing to update the commit of the brightray submodule, the brightray_example bootstrap script needs to be updated to call the brightray bootstrap script. Pulling from the current electron build on my machine, that call was:

``` bash
./vendor/brightray/script/bootstrap \
  --target_arch x64 \
  -c 82751b122d7f5cbedee5c662acc8cd1f1be8036d \
  https://s3.amazonaws.com/github-janky-artifacts/libchromiumcontent
```

But the right way to handle that would be to port electron's bootstrap script over, since it determines the correct calling flags for you.

**Comment by pfrazee** [comment]
Per https://groups.google.com/a/chromium.org/forum/#!msg/chromium-dev/roY78iTblYc/bb8nYsxfCgAJ, scoped_ptr needs to change to std::unique_ptr

**Comment by pfrazee** [comment]
Getting nowhere fast. I'm taking a break. (There's a lot of outdated code here.)
